CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Library] Update color semantic tokens to better manage light and dark modes values
 - [Library] Update elevation colors focus light and dark
 - [Library] Rename space padding inline component tokens by removing "component" word
+- [Library] Improve documentation about raw tokens definitions
```

Comment:
je me permets de poser une question : cette ligne ne devrait-elle pas être associée à une issue ?

Reply:
You are right!